packages/blockchain-extension/extension/explorer/walletExplorer.ts

```diff
@@ -130,21 +139,28 @@ export class BlockchainWalletExplorerProvider implements BlockchainExplorerProvi
         return tree;
     }
 
-    private async populateWallets(walletRegistryEntries: FabricWalletRegistryEntry[]): Promise<Array<WalletTreeItem>> {
-        const tree: Array<WalletTreeItem> = [];
+    private async populateWallets(walletRegistryEntries: any[]): Promise<Array<BlockchainTreeItem>> {
+        const tree: Array<BlockchainTreeItem> = [];
         for (const walletRegistryEntry of walletRegistryEntries) {
-            // get identityNames in the wallet
-            const walletGenerator: IFabricWalletGenerator = FabricWalletGeneratorFactory.getFabricWalletGenerator();
-            const wallet: IFabricWallet = await walletGenerator.getWallet(walletRegistryEntry);
-            const identityNames: string[] = await wallet.getIdentityNames();
+            // if we've got an array then it's a group of wallets, so push a WalletGroupTreeItem instead
+            if (walletRegistryEntry.length) {
```

Comment:
I'm assuming you're checking if 'walletRegistryEntry' is an array?
Probably best just calling `array. isArray()` if you want to do this.
If so, it probably doesn't make sense for these variables to be called `walletRegistryEntry` and `walletRegistryEntries`.